src/adapters/uni-v3/adapter.ts

```diff
@@ -0,0 +1,125 @@
+import { Decimal } from '../../utils/numerics';
+import {
+  DecodedOrder,
+  DecodedStrategy,
+  EncodedStrategy,
+} from '../../common/types';
+import { UniV3CastStrategy, UniV3Pool, UniV3Position } from './types';
+import { decodeStrategy } from '../../strategy-management/utils';
+import { encodeOrder } from '../../utils/encoders';
+
+/**
+ * Constants for Uniswap V3 calculations
+ */
+const TICK_BASE = new Decimal('1.0001');
```

Comment:
You can use `TICK_BASE_LN = Decimal.ln('1.0001')` instead of recalculating it every time the function is called.

Reply:
applying ty